### PR TITLE
Clicking a hunk now always scrolls to the correct position

### PR DESCRIPTION
**Pull request by txsmith**

### CODE REVIEW

File: src/projects/diff/group.html [hunk 1]
```diff
@@ -228,14 +229,10 @@
         }
       },
       _groupChanged: function(group, editable, githubUser) {
+        console.log('reset', group, editable, githubUser);
```
Comment (TimvdLippe) [inline src/projects/diff/group.html]: Left-over debug statement.

File: src/projects/diff/highlight.html [hunk 2]
```diff
@@ -69,7 +69,7 @@
       }
       pre {
         margin: 0;
-        line-height: 19px;
+        line-height: 18px;
```
Comment (TimvdLippe) [inline src/projects/diff/highlight.html:72]: :joy:

File: src/projects/diff/group.html [hunk 3]
```diff
@@ -228,14 +229,10 @@
...
         this.$.hunks.reset(function(hunk) {
           return this._createDraggableHunk(hunk, editable, githubUser);
         }.bind(this));
-        this.async(function() {
-          // Force a recalc to make sure that the `:empty` pseudo selector
-          // is correctly cleared after stamping groups to the hunks section
-          this.$.hunks.style.zoom = this.$.hunks.style.zoom ? '0.999999' : '1';
```
Comment (TimvdLippe) [inline src/projects/diff/group.html:237]: :fire:

File: src/projects/diff/diff.html [hunk 4]
```diff
@@ -176,7 +176,7 @@
       },
       jumpToHunk: function(groupId, hunkId) {
         var scrollTop = Math.min(
-                          this.$$('#group-'+groupId).jumpToHunk(hunkId) - 40,
+                          this.$$('#group-'+groupId).jumpToHunk(hunkId) - 96 - 16,
```
Comment (TimvdLippe) [inline src/projects/diff/diff.html]: We might want to extract a constant for this to state why these values.

**Comment by eanker** [comment]
LGTM

**Pull request merged**